lib/mosaic-interactions/src/main/java/org/eclipse/mosaic/interactions/environment/LidarUpdates.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2024 Fraunhofer FOKUS and others. All rights reserved.
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contact: mosaic@fokus.fraunhofer.de
+ */
+
+package org.eclipse.mosaic.interactions.environment;
+
+import static org.apache.commons.lang3.builder.ToStringStyle.SHORT_PREFIX_STYLE;
+
+import org.eclipse.mosaic.lib.spatial.PointCloud;
+import org.eclipse.mosaic.rti.api.Interaction;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class LidarUpdates extends Interaction {
+    private static final long serialVersionUID = 1L;
+
+    /**
+     * String identifying the type of this interaction.
+     */
+    public final static String TYPE_ID = createTypeIdentifier(LidarUpdates.class);
+
+    /**
+     * Time at which the next sensor update will be sent.
+     */
+    private long nextUpdate;
+
+    /**
+     * List of {@link PointCloud} containing LiDAR data from the simulator.
+     */
+    private Map<String, PointCloud> updated = new HashMap<>();
+
+    public LidarUpdates(long time, Map<String, PointCloud> updated) {
+        super(time);
+        this.updated = updated;
+    }
+
+    public Map<String,PointCloud> getUpdated() {
+        return this.updated;
+    }
+
+    public void setNextUpdate(long nextUpdate) {
+        this.nextUpdate = nextUpdate;
+    }
+
+    @Override
+    public int hashCode() {
+        return new HashCodeBuilder(5, 17)
+                .append(nextUpdate)
+                .append(updated)
+                .toHashCode();
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (obj == null) {
+            return false;
+        }
+        if (obj == this) {
+            return true;
+        }
+        if (obj.getClass() != getClass()) {
+            return false;
+        }
+
+        LidarUpdates other = (LidarUpdates) obj;
+        return new EqualsBuilder()
+                .append(this.nextUpdate, other.nextUpdate)
+                .append(this.updated, other.updated)
+                .isEquals();
+    }
+
+    //TODO what does this do?
+    @Override
+    public String toString() {
+        return new ToStringBuilder(this, SHORT_PREFIX_STYLE)
+                .appendSuper(super.toString())
+                .append("updated", this.updated)
```

Comment:
Don't just append the map here. I'm not even sure what happens if you just put the map here, I guess it would just print `HashMap@a78f8e9b` or something like that. Better append something meaningful, such as 

```java
  return new ToStringBuilder(this, SHORT_PREFIX_STYLE)
                .appendSuper(super.toString())
                .append("updated", "Updated Lidar data for vehicles [" + /*TODO collect vehicle ids*/ + "]);